docker/compose/indexer-postgres/docker-compose.yaml

```diff
@@ -0,0 +1,71 @@
+version: "3.8"
+services:
+  db:
+    container_name: postgres
+    image: postgres
+    restart: always
+    volumes:
+      - aptos-indexer-db:/var/lib/postgresql/data
+    env_file:
+      - ./.env
+    environment:
+      POSTGRES_USER: ${POSTGRES_USER}
+      POSTGRES_PASSWORD: ${POSTGRES_PASSWORD}
+      POSTGRES_DB: ${POSTGRES_DB}
+    networks:
+      aptos-indexer:
+        ipv4_address: 172.16.3.10
```

Comment:
could we use names instead of IPs?

I suspect you followed this PR: https://github.com/aptos-labs/aptos-core/commit/9aca522c6f21cf924313c6fab37a8c1bec5b7cd0

We need to revert this as well.

CC: @rustielin

Reply:
Yea we can try using DNS names. also cc @geekflyer for test coverage